### PR TITLE
Added resources from my Master Lecture

### PR DESCRIPTION
**Pull request by Joerg-Schultz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Awesome Bioinformatics (WUBSyB Version)
+﻿# Awesome Bioinformatics (WUBSyB Version)
```
Comment (iimog) [inline README.md:1]: The difference in this line is that a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) has been added. This was probably done automatically by the text editor. Just out of curiosity: which editor do you use @Joerg-Schultz ?
The BOM is optional and does not hurt, so in my opinion we can keep it.

File: README.md [hunk 2]
```diff
@@ -10,10 +10,20 @@
 
 ## Videos 🎥
 
+- [Pavel Pevzner explains short read assembling] (https://www.youtube.com/watch?feature=player_embedded&list=PLQ-85lQlPqFNGdaeGpV8dPEeSm3AChb6L&v=vjB6nhOu3BY)
```
Comment (iimog) [inline README.md:13]: the part `feature=player_embedded&list=PLQ-85lQlPqFNGdaeGpV8dPEeSm3AChb6L&` can be removed

**Review by iimog — APPROVED** [review]
Looks good to me. Added two optional minor comments.

**Comment by konrad** [comment]
Many Thanks @Joerg-Schultz 🎉

**Pull request merged**